client/scss/components/_header.scss

```diff
@@ -56,6 +56,9 @@ header {
     // stylelint-disable-next-line property-disallowed-list
     left: 0.5em;
     inset-inline-start: 0.5em;
+
+    // Icon Positioning (Vertically Centered)
+    padding: 10px 0;
```

Comment:
**Suggestion:** We could also align vertically using 50% of the height of the element to ensure they will always be centered, even if the input was to change in size a bit. I'd suggest changing `top: 0.3em;` to be: 

```
    top: 50%;
    transform: translateY(-50%);
```

Then we wouldn't need to adjust the padding at all.